draft-ietf-jsonpath-base.md

```diff
@@ -1224,6 +1212,11 @@ The `filter-selector` works with arrays and objects exclusively. Its result is a
 A relative path, beginning with `@`, refers to the current array element or member value as the
 filter selector iterates over the array or object.
 
+A member or element value by itself in a Boolean context is
```

Comment:
That doesn't work for me.
Once something is a value, it is that value; no further conversion happens.
This sentence was originally about queries that deliver a node list.
A value that doesn't exist isn't a value, no?

Reply:
Yes, I've added a commit with better wording.